apps/services/auth-public-api/src/app/modules/delegations/meDelegations.controller.ts

```diff
@@ -123,28 +109,28 @@ export class MeDelegationsController {
   @Scopes(AuthScope.readDelegations)
   @FeatureFlag(Features.customDelegations)
   @Get(':delegationId')
-  @ApiOperation({
+  @Documentation({
+    response: { status: 200, type: DelegationDTO },
     description: `Finds a single delegation by ID where the authenticated user is either giving or receiving.
        Does not include delegations from NationalRegistry or CompanyRegistry.`,
+    params: [
+      {
+        name: 'delegationId',
+        type: 'string',
+        description: 'Delegation ID.',
+      },
+    ],
```

Comment:
What do you think about saving more space for param/query by doing this:

```
params: {
  delegationId: {
    type: 'string',
    description: 'Delegation ID.',
  }
}
```

I know you're re-using the @nestjs/swagger type, but they could probably still be used like this:

```
params: Record<string, Omit<ApiParamOptions, 'name'>>,
queries: Record<string, Omit<ApiQueryOptions, 'name'>>,
```

Reply:
Definitely..  agree with that.